cob_hardware_config/cob4-4/config/base_joint_limits.yaml

```diff
@@ -1,73 +1,31 @@
 joint_limits:
   fl_caster_rotation_joint:
-    has_position_limits: false
-    min_position: -3.14
-    max_position: 3.14
     has_velocity_limits: true
     max_velocity: 15.0
-    has_acceleration_limits: false
-    max_acceleration: 2.0
-    has_jerk_limits: false
-    max_jerk: 100.0
-    has_effort_limits: false
-    max_effort: 5.0
+    #has_acceleration_limits: false
+    #max_acceleration: 2.0
```

Comment:
I commented out the lines because this feature is still not supported,  @ipa-mdl also suggested it https://github.com/ipa320/cob_robots/pull/332#discussion_r40362904